test/e2e/network/dns_configmap.go

```diff
@@ -160,6 +162,7 @@ func (t *dnsNameserverTest) run() {
 	defer t.deleteDNSServerPod()
 
 	if t.name == "coredns" {
+		originalConfigMapData = t.fetchCoreDNSConfigMapData()
 		t.setConfigMap(&v1.ConfigMap{Data: map[string]string{
```

Comment:
As we expect that the Configmap returned to the original one, WHATEVER the result of the test. We need to ensure that it is setback to the "originalConfigMapData" with a "defer function" (like the one of line 162).